ppa-update-tools/README.md

```diff
@@ -0,0 +1,19 @@
+# Ppa update tools
+
+This directory contains programs that help test checkbox PPAs.
+
+## ppa_version_published.py
+
+This program checks whether a specified package is available in launchpad
+for a the PPA correspondent to a snap channel
+
+Characteristics include package name, deb name, ubuntu versions and 
```

Comment:
For most of the cases, the package name and the deb name are the same, but for `checkbox-support` the package is called  `checkbox-support` and the `.deb` file is called `python3-checkbox-support_3.3.0`

- http://ppa.launchpad.net/checkbox-dev/edge/ubuntu/pool/main/c/checkbox-support/
- http://ppa.launchpad.net/checkbox-dev/edge/ubuntu/pool/main/c/checkbox-support/python3-checkbox-support_3.3.0~dev96~ubuntu23.10.1_all.deb

That's why I added two different names

Reply:
Should I keep it as it is? Should I add this field only for the package that has a different name than the `.deb` file?

Reply:
So can we have only one of them? in this case I guess deb name

Reply:
package is called "python3-checkbox-support", not "checkbox-support"

Reply:
Yes, that's right, technicaly it would be the "source" name instead of "package" name. 
We would still need both, because in the case of checkbox-support, the *source* is named `checkbox-support`, but the  (deb) *package* is named `python3-checkbox-support`

I think the correct naming should be:
   source: checkbox-support
   package: python3-checkbox-support
   versions: [ 18.04, 20.04, 22.04, 23.04, 230, 24.04 ]
   architectures: [ all ]

Reply:
+1